apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -516,7 +516,7 @@
             }
 
             return accessURL.replaceAll(Pattern.quote("${UserTenantHint}"), userTenantHint)
-                            .replaceAll(Pattern.quote("/t/carbon.super/"), "/");
+                            .replaceAll(Pattern.quote("/t/carbon.super"), "");
```

Comment:
Shouldn't we use the same utility class here as well?

Reply:
Will update it after merging this https://github.com/wso2/carbon-identity-framework/pull/3388